src/main/scala/Day1/Basic.scala

```diff
@@ -0,0 +1,92 @@
+package Day1
+
+object Basic extends App {
+
+  // Hello World!
+  println("Hello World!")
+
+  // Assignment
+  var helloWorld = "Hello World!"
+  println(helloWorld)
+
+  // Parameters
+  def printWord(word: String): Unit = println(word)
+  printWord("test")
+
+  // Return types
+  def printHelloWorld() = "Hello World!"
+  println(printHelloWorld())
+
+  // Type Inference
+  def returnAnyType(a: Any): Unit = println(a)
+  returnAnyType(2.003)
+  returnAnyType(23)
+  returnAnyType(true)
+  returnAnyType("String")
+
+  // Strings
+  def stringPrinting(str: String, num: Int): Unit =
+    println(str.takeRight(num))
+
+  stringPrinting("Hello",3)
+
+  // Strings 2
+  def charReplace(str1: String, str2: String, char1: Char, char2: Char): Unit ={
+    val fullString = str1 + str2
+    println(fullString.replace(char1,char2))
```

Comment:
you don't need the fullString variable could just do it like this:
`
println(str1.concat(str2).replace(char1,char2))
`